msmbuilder/cluster/agglomerative.py

```diff
@@ -227,3 +275,4 @@ def fit_predict(self, X):
 
 class LandmarkAgglomerative(MultiSequenceClusterMixin, _LandmarkAgglomerative, BaseEstimator):
```

Comment:
I'm not sure- I didn't change anything from @rmcgibbo's preexisting `LandmarkAgglomerative` that wasn't necessary to accommodate the Ward stuff.